config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -121,14 +121,11 @@ periodics:
       - "./scripts/ci-e2e.sh"
       env:
       - name: GINKGO_SKIP
```

Comment:
Bumped mgmt cluster version of e2e-mink8s to v1.23 as new minimum mgmt cluster version is v1.23 now

This also means we don't have to skip ClusterClass tests anymore as ClusterClass is supported on v1.23

Reply:
Was wondering about ClusterClass change, found it now here )